### ISSUE TITLE
Failed to run artifactory-cleanup-0.4.1 command

### ISSUE THREAD

**Issue by shenxianpeng** [devopshq/artifactory-cleanup#64]
Hi,

When I try to run the command `artifactory-cleanup --help` or `artifactory-cleanup --user myuser --password mypassword --artifactory-server https://myrepo.com/artifactory --config myreponame.py` have the following error. 

```bash
Traceback (most recent call last):
  File "/home/ubuntu/.local/bin/artifactory-cleanup", line 5, in <module>
    from artifactory_cleanup.artifactorycleanup import ArtifactoryCleanupCLI
ImportError: cannot import name 'ArtifactoryCleanupCLI' from 'artifactory_cleanup.artifactorycleanup' (/home/ubuntu/.local/lib/python3.8/site-packages/artifactory_cleanup/artifactorycleanup.py)
```

I'm not sure if I miss something, could you help me with some advice? 

Here is our local environment:

```bash
$ pip show artifactory-cleanup
Name: artifactory-cleanup
Version: 0.4.1
Summary: Rules and cleanup policies for Artifactory
Home-page: https://github.com/devopshq/artifactory-cleanup
Author: Alexey Burov
Author-email: allburov@gmail.com
License: MIT
Location: /home/ubuntu/.local/lib/python3.8/site-packages
Requires: attrs, dohq-artifactory, hurry.filesize, plumbum, prettytable, teamcity-messages, treelib
Required-by: 

$ python3 -V
Python 3.8.10
```

**Comment by shenxianpeng** [comment]
Installed  `artifactory-cleanup-0.4.0` works.
```
Successfully installed artifactory-cleanup-0.4.0
ubuntu@CN-L-2680:~/u2cicd$  artifactory-cleanup --help
Usage:
    artifactory-cleanup [SWITCHES]

Meta-switches:
    -h, --help                          Prints this help message and quits
    --help-all                          Prints help messages of all sub-commands and quits
    -v, --version                       Prints the program's version and quits

Switches:
    --artifactory-server VALUE:str      URL to artifactory, e.g: https://arti.example.com/artifactory; required
    --config VALUE:str                  Name of config with list of policies
    --days-in-future VALUE:str          Simulate future behaviour; excludes --destroy
    --debug                             Only print artifacts that can be deleted with the specified cleaning policies and rules
    --destroy                           Remove artifacts
    --password VALUE:str                Password to access to the artifactory; required
    --policy-name VALUE:str             Name for a rule
    --user VALUE:str                    Login to access to the artifactory; required
```

**Comment by allburov** [comment]
My fault! 
Fixed in 0.4.2, please try update to the new version

**Comment by shenxianpeng** [comment]
It works in `0.4.2`
